app/assets/stylesheets/themes/light/homePage/conversation.less

```diff
@@ -57,9 +57,10 @@
         }
 
         background: white;
-        padding: 0px;
+        padding: 15px;
         margin-bottom: @vertical-padding;
-        .card-dark();
+        border-top: 1px solid @border-dark-color;
+        border-bottom: 1px solid @border-dark-color;
```

Comment:
Same reason as above, by remove card-dark I have no way to potentially add back shadows or maybe some other style.